gcp/modules/k8s-snapshots/main.tf

```diff
@@ -0,0 +1,45 @@
+terraform {
+  backend "gcs" {}
+}
+
+variable "prd_deltas" {
```

Comment:
Per-environment settings should be set in `live/dev`, `live/stg`, etc.

...but I don't think these settings belong in this module. See below.

Reply:
Removed.